compiler/src/iree/compiler/Tools/init_mlir_dialects.h

```diff
@@ -67,6 +68,7 @@ inline void registerMlirDialects(DialectRegistry &registry) {
                   shape::ShapeDialect>();
   // clang-format on
   tensor::registerInferTypeOpInterfaceExternalModels(registry);
+  tensor::registerTilingOpInterfaceExternalModels(registry);
```

Comment:
just found that it should be `registerTilingInterfaceExternalModels`. :p

Reply:
Haha! Yeah! Haven't touched this branch in a while